e2e-tests/docs/CI.md

```diff
@@ -0,0 +1,135 @@
+# CI Steps and Processes for Testing
+
+This document provides a comprehensive overview of our Continuous Integration (CI) testing processes. It explains when and how different test jobs are triggered, where to monitor them, and details about the environments hosting these tests, along with guidance on how to access them.
+
+## GitHub Pull Requests
+
+When a new Pull Request (PR) is opened at [backstage-showcase](https://github.com/janus-idp/backstage-showcase), tests can be triggered in two ways:
```

Comment:
PR tests are triggered automatically if changes include code changes that affect tests. For example, doc only changes will skip the tests. That said, do you think we should mention when we need to put '/ok-to-test'? it's not always necessary to add it.

Reply:
hi! just edited it and remarked this on _Commenting `/ok-to-test`_. also added the link from Joseph's comment for further reference. Thanks!